src/main/java/de/metas/ui/web/handlingunits/process/WebuiHUTransformCommand.java

```diff
@@ -253,11 +253,12 @@ private WebuiHUTransformCommandResult action_updatePlanningReceiptOwnerPM(final
 	 */
 	private WebuiHUTransformCommandResult action_SplitCU_To_ExistingTU(final HUEditorRow cuRow, final I_M_HU tuHU, final BigDecimal qtyCU)
 	{
-		newHUTransformation().cuToExistingTU(cuRow.getM_HU(), qtyCU, tuHU);
+		final List<I_M_HU> createdCUs = newHUTransformation().cuToExistingTU(cuRow.getM_HU(), qtyCU, tuHU);
 
 		return WebuiHUTransformCommandResult.builder()
 				.huIdChanged(cuRow.getM_HU_ID())
 				.huIdChanged(tuHU.getM_HU_ID())
+				.huIdsToAddToView(createdCUs.stream().map(hu -> hu.getM_HU_ID()).collect(ImmutableList.toImmutableList()))
```

Comment:
not sure this is OK because the CUs might not be top levels and in that case it's not OK.
For those which are top level CUs... it's OK.

But in your case, as I understand, you are adding some CUs to an existing TU, so i think your "createdCUs" are not top levels.